ccm_web/server/src/api/api.controller.ts

```diff
@@ -19,4 +22,24 @@ export class APIController {
   getGlobals (@Session() session: SessionData): Globals {
     return this.apiService.getGlobals(session)
   }
+
+  @Get('course/:id/name')
+  async getCourseName (
+    @Param('id', ParseIntPipe) courseId: number, @Session() session: SessionData
+  ): Promise<string> {
+    const { userLoginId } = session.data
+    const result = await this.apiService.getCourseName(userLoginId, courseId)
+    if (result === null) throw new InternalServerErrorException('Error occurred while communicating with Canvas')
+    return result
+  }
+
+  @Post('course/:id/name')
```

Comment:
Updating a courseName is actually a PUT call, can we change it with same HTTP method?

Reply:
Sure, `PUT` does seem more correct

Reply:
Done. See https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/121/commits/2ee56331c71ff1d82bad4548ddf9a4f73fd9626c